pkg/controller/backup_storage_location_controller.go

```diff
@@ -55,7 +55,7 @@ type BackupStorageLocationReconciler struct {
 func (r *BackupStorageLocationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	log := r.Log.WithField("controller", BackupStorageLocation)
 
-	log.Info("Checking for existing backup locations ready to be verified; there needs to be at least 1 backup location available")
+	log.Info("Verifying availablity of backup storage locations.")
```

Comment:
Given how often this runs, I think we should bump this down to debug. Otherwise, it just files the logs up, which makes parsing tough.